ocaml/xapi-types/jbuild

```diff
@@ -30,13 +30,13 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
   (public_name xapi-types)
   (flags (:standard %s))
   (libraries (
-   xapi-consts
-   rpclib
+   astring
+   http-svr
    ppx_deriving_rpc
-   stdext
+   rpclib
    threads
-   http-svr
-   uuid
+   uuidm
+   xapi-consts
```

Comment:
Just curiosity here: does the order make a difference?

Reply:
No, I've sorted them alphabetically only because I find it easier to find what I need with a glance :P